pkgs/by-name/ko/koruri/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "koruri";
+  version = "20210720";
+
+  src = fetchFromGitHub {
+    owner = "Koruri";
+    repo = "Koruri";
+    rev = "6e173461858616f7b701b8e297b3e408ae224c7f";
+    hash = "sha256-zL9UtT15mWvsXgGJqbTs6cOsQaoh/0AIAyQ5z7JpTXk=";
+  };
+  installPhase = ''
+    runHook preInstall
+    install -Dm644 *.ttf -t $out/share/fonts/${pname}
```

Comment:
```suggestion
    install -Dm644 *.ttf -t $out/share/fonts/koruri
```

Reply:
What is the reason for such a correction?

Reply:
See #277994 for reasoning.

Reply:
Resolved by: bf1428225f7955c5d26f3a54f9c7c154b18ce24a
Thank you for writing the reason.